opm/core/wells/WellsManager.cpp

```diff
@@ -200,6 +200,7 @@ namespace Opm
     /// Construct wells from deck.
     WellsManager::WellsManager(const Opm::EclipseState& eclipseState,
                                const Opm::Schedule& schedule,
+                               const SummaryState& summaryState,
                                const size_t timeStep,
```

Comment:
What is your plan with adding summarystate to the WellsManager?

Reply:
![image](https://user-images.githubusercontent.com/1321665/58861621-0e5ebf80-86af-11e9-9e97-caeb12c70343.png)

Reply:
Hehe - seriously; the `SummaryState` is required context to evaluate dynamic (i.e. UDA based) controls.